perfkitbenchmarker/vm_util.py

```diff
@@ -256,7 +256,7 @@ def WrappedFunction(*args, **kwargs):
 
 
 def IssueCommand(cmd, force_info_log=False, suppress_warning=False,
-                 env=None, timeout=DEFAULT_TIMEOUT, input=None):
+                 env=None, timeout=DEFAULT_TIMEOUT, cwd=None, use_shell=False):
```

Comment:
From https://docs.python.org/2/library/subprocess.html#frequently-used-arguments
"the use of shell=True is strongly discouraged in cases where the command string is constructed from external input"

Can we try and avoid it ? Otherwise we are opening a security loophole in pkb